moviepy/video/io/ImageSequenceClip.py

```diff
@@ -5,7 +5,7 @@
 import os
 
 import numpy as np
-from imageio import imread
+from imageio.v3 import imread
```

Comment:
```suggestion
import imageio.v3 as iio
```

Same here, I would recommend to import the entire namespace. This will give you access to some niceties (see below).